custom_webshop_description/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+{
+    'name' : 'Extended Webshop Description',
+    'description' : """
+        This module adds an "Ecommerce Extended Description" field to the product template.
+        - Supports translations.
+        - Displays on the product page in eCommerce.
+        - Included in export/import functionality.
+    """,
+    'version' : '0.1',
+    'category' : 'Website',
+    'author' : 'Sudhirkumar Sharma',
+    'website': "https://www.odoo.com",
+    'depends' : [
+        'base',
+        'website_sale',
+    ],
```

Comment:
```suggestion
    'depends' : [
        'website_sale',
    ],
```
I think it should work

Reply:
removed and its working fine